src/main/java/org/cryptomator/common/mount/MountWithinParentUtil.java

```diff
@@ -99,16 +119,22 @@ private static void waitForMountpointRestoration(Path mountPoint) throws FileAlr
 		}
 	}
 
-	private static void checkIsDirectory(Path toCheck) throws MountPointPreparationException {
+	private static void checkIsDirectory(Path toCheck) throws IllegalMountPointException {
```

Comment:
the method name should reflect, that this method is intended for mount points

Reply:
Resolved by 1c34402c87077252871cea2cbf4ec5f546adcb4d.